### DISCUSSION TITLE
Attributes table

### DISCUSSION THREAD

**Discussion by szapoun** [crystalyragui/MARC2RDA#544] [General]
Created this new issue. If I should have used another issue, please tell me. 

[Attributes table in Google Drive](https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=1796318524#gid=1796318524)

I have reviewed some of the mappings, but I would like to discuss the following, if possible. 

rdawd:P10088 has title of work --> there are 4 different mappings. Propose to merge into one --> $t + $n$p (any of the referred subfields in the order they appear in the field) 

$m --> has medium of performance of musical content of representative expression --> Question: Do not know about cataloguing of musical works.  This is an expression property. How do we one that $k is the key of the representative expression? If not, we mint work and expression in order to assign key? It seems too much to me. 

$o --> not mapped. But similar case to $m. or note on work?

$r --> similar case to $m

$n --> check together proposed mappings. Some seem odd to me. 

$v --> 8XX. It is proposed to be mapped as numbering of part, but I disagree. I think $n can be mapped with this property

$x --> ISSN

**Comment by sarahruska** [comment]
Tagging @CECSpecialistI since it looks like this was meant for her rather than the other Crystal tagged :)

Looks like this is related to #448 

Adding to our current BSR milestone

**Comment by cspayne** [comment]
- X00 ind1 = 0 $a and $b (row 2) - should this be for ind1 = 0 or 1, or is it only for ind1 - 0? Should it be mapped if there is no $b? 

- X00 ind1 = 0 or 1 $d (row 5) - should this only be mapped if the value cannot be mapped to any of the other narrower options? Or is it mapped for each $d present alongside narrower properties such as date of birth, date of death, etc.

**Comment by cspayne** [comment]
- $f when 040 $e = rda is mapped as has date of expression, but we only mint related works, not expressions

**Comment by sarahruska** [comment]
> * X00 ind1 = 0 or 1 $d (row 5) - should this only be mapped if the value cannot be mapped to any of the other narrower options? Or is it mapped for each $d present alongside narrower properties such as date of birth, date of death, etc.

Examples shared from today's meeting:
- @AdamSchiff's jin shi example: `100 1 Bao, Rong, ǂd jin shi 809`
- @JianPLee's ju ren example: `Chen, Denglong, $d ju ren 1774`
- @AdamSchiff's 'approximately' examples:
    - `Aaron, ǂc of Zhitomir, ǂd -approximately 1817`
    - `Aaron, W. F., ǂd active approximately 1860`
    - `Abate, Nicolò dell', ǂd approximately 1509-1571`
    - `ʻAbbās ibn ʻAbd al-Muṭṭalib, ǂd approximately 566-approximately 653`
    - `ʻAbd al-ʻAzīz Muḥammad, ǂd 1866 or 1867-approximately 1948`

**Comment by cspayne** [comment]
Output based on above discussions: 

```xml
<rdf:Description rdf:about="http://marc2rda.edu/fake/relator-test9age#d22e158">
      <fake:marcfield>F100 1# $a Bao, Rong, $d jin shi 809</fake:marcfield>
      <rdf:type rdf:resource="http://rdaregistry.info/Elements/c/C10004"/>
      <rdaad:P50377>Bao, Rong, jin shi 809</rdaad:P50377><!--rdaad:P50377 = has access point for person-->
      <rdaad:P50098>809</rdaad:P50098><!--rdaad:P50098 = has period of activity of person-->
   </rdf:Description>
   <rdf:Description rdf:about="http://marc2rda.edu/fake/relator-test9age#d22e163">
      <fake:marcfield>F700 1# $a Chen, Denlong, $d ju ren 1774</fake:marcfield>
      <rdf:type rdf:resource="http://rdaregistry.info/Elements/c/C10004"/>
      <rdaad:P50377>Chen, Denlong, ju ren 1774</rdaad:P50377><!--rdaad:P50377 = has access point for person-->
      <rdaad:P50098>1774</rdaad:P50098><!--rdaad:P50098 = has period of activity of person-->
   </rdf:Description>
   <rdf:Description rdf:about="http://marc2rda.edu/fake/relator-test9age#d22e168">
      <fake:marcfield>F700 1# $a Aaron, $c of zhitomir, $d -approximately 1817</fake:marcfield>
      <rdf:type rdf:resource="http://rdaregistry.info/Elements/c/C10004"/>
      <rdaad:P50377>Aaron, of zhitomir, -approximately 1817</rdaad:P50377><!--rdaad:P50377 = has access point for person-->
      <rdaad:P50120>approximately 1817</rdaad:P50120><!--rdaad:P50120 = has date of death-->
   </rdf:Description>
   <rdf:Description rdf:about="http://marc2rda.edu/fake/relator-test9age#d22e175">
      <fake:marcfield>F700 1# $a Aaron, W.F., $d active approximately 1860</fake:marcfield>
      <rdf:type rdf:resource="http://rdaregistry.info/Elements/c/C10004"/>
      <rdaad:P50377>Aaron, W.F., active approximately 1860</rdaad:P50377><!--rdaad:P50377 = has access point for person-->
      <rdaad:P50098>approximately 1860</rdaad:P50098><!--rdaad:P50098 = has period of activity of person-->
   </rdf:Description>
   <rdf:Description rdf:about="http://marc2rda.edu/fake/relator-test9age#d22e180">
      <fake:marcfield>F700 1# $a Abate, Nicolò dell', $d approximately 1509-1571</fake:marcfield>
      <rdf:type rdf:resource="http://rdaregistry.info/Elements/c/C10004"/>
      <rdaad:P50377>Abate, Nicolò dell', approximately 1509-1571</rdaad:P50377><!--rdaad:P50377 = has access point for person-->
      <rdaad:P50121>approximately 1509</rdaad:P50121><!--rdaad:P50121 = has date of birth-->
      <rdaad:P50120>1571</rdaad:P50120><!--rdaad:P50120 = has date of death-->
   </rdf:Description>
   <rdf:Description rdf:about="http://marc2rda.edu/fake/relator-test9age#d22e185">
      <fake:marcfield>F700 1# $a ʻAbbās ibn ʻAbd al-Muṭṭalib, $d approximately 566-approximately 653</fake:marcfield>
      <rdf:type rdf:resource="http://rdaregistry.info/Elements/c/C10004"/>
      <rdaad:P50377>ʻAbbās ibn ʻAbd al-Muṭṭalib, approximately 566-approximately 653</rdaad:P50377><!--rdaad:P50377 = has access point for person-->
      <rdaad:P50121>approximately 566</rdaad:P50121><!--rdaad:P50121 = has date of birth-->
      <rdaad:P50120>approximately 653</rdaad:P50120><!--rdaad:P50120 = has date of death-->
   </rdf:Description>
   <rdf:Description rdf:about="http://marc2rda.edu/fake/relator-test9age#d22e190">
      <fake:marcfield>F700 1# $a ʻAbd al-ʻAzīz Muḥammad, $d 1866 or 1867-approximately 1948</fake:marcfield>
      <rdf:type rdf:resource="http://rdaregistry.info/Elements/c/C10004"/>
      <rdaad:P50377>ʻAbd al-ʻAzīz Muḥammad, 1866 or 1867-approximately 1948</rdaad:P50377><!--rdaad:P50377 = has access point for person-->
      <rdaad:P50121>1866 or 1867</rdaad:P50121><!--rdaad:P50121 = has date of birth-->
      <rdaad:P50120>approximately 1948</rdaad:P50120><!--rdaad:P50120 = has date of death-->
   </rdf:Description>
```

**Comment by AdamSchiff** [comment]
To which could be added other examples:

A. Z., ǂd approximately 1870-
Achilles Painter, ǂd active approximately 450 B.C.-425 B.C.
ʻAlī, ibn al-Shāh al-Fārisī, ǂd active approximately 13th century
Barnard, John, ǂd active 1641
Braccesco, Giovanni, ǂd 1481?-approximately 1555
Blo-gros-rin-chen-seṅ-ge, ǂc Kun-mkhyen, ǂd active 14th century-15th century
Chen, Bin, ǂd 1968 June 26-
Chen, Chao, ǂd 1897-1963?
Allen, Ron, ǂd 1947 September 13-2010
Cheng, Shuyao, ǂd 1917 November-1993 September
Ellefsen, Anette, ǂd 1980 March 16-1980 September 4
Drysdale, William, ǂd -after 1791 September 5
González, Bobby, ǂd -2023 September 21
Bao, Rong, ǂd approximately 689-not before 743
Dmitriev, Nikolaĭ Ivanovich, ǂd 1880-not before 1946
Ghazālī, Muḥammad, ǂd active before 1714
Lawson, Anthony, ǂd before 1729 January 20-1815
Mosman, Jan Ludolf, ǂd before 1685?-1731
Spalenza, Pietr'Antonio, ǂd approximately 1545-before 1577
Arconati, Felice Antonio, ǂd 1613?-after 1689
Coxsun, Robert, ǂd 1489 or 1490-after 1548

I think we should map to the specific properties (date of birth, date of death, period of activity of person) and not also map to the broader related timespan of person.  I can't see any case where the value in $d can't be mapped to one of the more specific properties.

If a value appears in front of a hyphen, it is a date of birth unless the word "active" or a term like "jin shi" or "ju ren" appears.  The latter are periods of activity.
A value after a hyphen is a date of death, unless the word "active" appears.
If there is no hyphen in $d, the date must be a period of activity.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by tmqdeborah** [comment]
Replying to https://github.com/uwlib-cams/MARC2RDA/issues/471#issuecomment-2608763019

> I think we should map to the specific properties (date of birth, date of death, period of activity of person) and not also map to the broader related timespan of person.  I can't see any case where the value in $d can't be mapped to one of the more specific properties.

I wonder whether the decision to map to _related timespan of person_ was intended to map only the date, without other terms, to a Timespan IRI?

> If there is no hyphen in $d, the date must be a period of activity.

I'm not sure about that, it could be an error for a birth date or death date or ... 

So, I would suggest using _related timespan of person_ instead, if there is no hyphen in $d.

**Comment by cspayne** [comment]
As this gets updated, we need to ensure that the relevant field templates in the code are updated!

**Comment by CECSpecialistI** [comment]
I'd prefer to avoid minting timespans at this point in our progress, and not predict errors. I agree with Adam's proposed approach.

**Comment by cspayne** [comment]
> I'd prefer to avoid minting timespans at this point in our progress, and not predict errors. I agree with Adam's proposed approach.

We wouldn't need to mint a timespan for "related timespan of person", it would be a string value. The question is whether dates that don't fit the cases we have already identified get mapped as period of activity or as related timespan of person.

**Comment by cspayne** [comment]
Is there a reason name of corporate body and name of family were not mapped in the attributes table but name of person is?

**Comment by AdamSchiff** [comment]
We can parse this out more carefully.  Period of activity will always be preceded by the word “active” or “jin shi” or “flourished” or “fl.”, etc., the exception being centuries.  But any century date should be a period of activity.  Some birth and death dates will also have words before them (e.g., “approximately”, “circa”, “ca.”) but should still have a hyphen.  There may be some older AACR dates without hyphens though: b. 1855 for example.  Or d. 1676. Not sure if OCLC cleaned all of these up in their database, but libraries could still have them in local files.

Adam, from an Uber in Mexico City 🙂

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Comment by cspayne** [comment]
I think the table may need some more work in regards to subfields for 130, 240, 630, 730, and 830, such as $a, $n, $p, $s mapping to title?

**Comment by CECSpecialistI** [comment]
@tmqdeborah would you follow up on this when you get back? Were the questions Cypress  asked in her last comment resolved or is there more work to do here?

**Comment by tmqdeborah** [comment]
Yes, I will follow up on this when I get back.

**Comment by CECSpecialistI** [comment]
Related issue on hold: https://github.com/uwlib-cams/MARC2RDA/issues/111

**Comment by CECSpecialistI** [comment]
Also: look at access point mapping table.

**Comment by tmqdeborah** [comment]
I am working on the https://github.com/uwlib-cams/MARC2RDA/issues/471 and I will add the attributes that can be mapped from https://github.com/uwlib-cams/MARC2RDA/issues/150and https://github.com/uwlib-cams/MARC2RDA/issues/111to that table.

**Comment by tmqdeborah** [comment]
I have split the [Heading Fields Attribute Mapping](https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=1796318524#gid=1796318524) into two:
- [Headings attributes for Agents](https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=580845826#gid=580845826)
- [Headings attributes for Works and Expressions](https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=1632134046#gid=1632134046)

The approach taken by this spreadsheet is to:
- Specify which of the provided mapping instructions from MARC fields/subfields  found in Headings fields to RDA elements apply to which types of works, expressions, and (occasionally) manifestations.
  - Transformation coding already uses the following labels for these different types of entities
    - "wor": Single expression work
    - "augWor":  Augmented work that is a single expression work
    - "aggWor":  Aggregating work
    - "relWor": Related work
    - “exp” : Expression of Single expression work
    - "augExp": Expression of Augmented work
    - “man”: Manifestation
  - These labels can be used to indicate when mapping instructions from MARC fields/subfields to RDA elements apply to which types of works, expressions, and manifestations

If this approach works for the coders, then I will expand it to cover the work and expression attributes taken from the body of the record, e.g., 334, 335, 336, 000, 008, etc.

All manifestation attributes apply to both aggregate and non-aggregate manifestations; so, the approach in this spreadsheet is not needed for them—the regular spreadsheets are sufficient.

**Useful links**
Access point mapping table: https://docs.google.com/spreadsheets/d/1CXBDgFvXzFiRytEz8G-9dkax8oyEA3q_/edit?gid=1248318886#gid=1248318886
Headings fields Personal names: https://docs.google.com/spreadsheets/d/1_wszM48Mrr2KTGsZgAboUVFVvAmJJm02/edit?gid=1691044164#gid=1691044164
Headings fields attribute mapping: https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=1796318524#gid=1796318524
Transformation coding: https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/transformationCode/m2r-relators.xsl#L1249
Transforming augmentations: https://docs.google.com/document/d/1tIGBUjSe4l3XZn88Naael-vUX4Aw2Hcc/edit	
MARC Bibliographic: https://www.loc.gov/marc/bibliographic/bdx11.html
MARC Bibliographic TOC: https://maint.loc.gov/marc/bibliographic/ecbdlist.html
RDA Registry: https://www.rdaregistry.info/Elements/

Please let me know if this approach makes sense. And please report errors and ask questions here.

**Comment by lake44me** [comment]
@tmqdeborah  Sorry, github won't let me delete the previous comment.   I'm looking at Headings Attributes for WE, appellations section. it would be helpful to understand what is meant by "if found, stop" e.g. in line 22.  Does this mean, if a Single Expression Work is found to have 100 ** $abcdjqu +g (if before $t) + 240 $adgknpt,  don't code for the property W: has AP/ AAP for work?

**Comment by lake44me** [comment]
I'm thinking there might be cases where it would be ok to code as an AP or AAP for the Work, say if there is only 240 $a, or $a, $n, $p .  Particcularly if it has a $0, 1, or $2 indicating the source...?    Not likely with 240 since those subfields were not available until recently.

**Comment by tmqdeborah** [comment]
@lake44me: in the "has AP/AAP for work" in rows 21-28  of [Heading Fields Attribute Mapping](https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=1632134046#gid=1632134046) under the "Headings attributes for WE" tab, Condition 1 is iterative for each column. That is:
- if 130 $adghknpt (**i.e., $a and any other qualifying subfields**) is found, stop; if not
- if 100 ** $abcdjqu +g (if before $t) + 240 $adgknpt is found, stop; if not
- if 110** $abcu + dgn (if before $t) + 240 $adgknpt, stop; if not
- 111** $acequ + dgn (if before $t) + 240 $adgknpt, stop;
- if not
- etc. until you reach row 28, the final possibility "245$anp" 

Note that "$a and any other qualifying subfields" or '$t any any other qualifying subfields' apply for all lists of subfields conditions.

Thanks for asking for clarification.

It would not be OK to code for only 240 without a 100-111; this should never occur; i.e., a 240 without an 100-111 is a 130.

**Comment by lake44me** [comment]
so "stop" means, code it and then stop?

**Comment by tmqdeborah** [comment]
By "stop" I mean do not run the next bit of code in that group of codes. I have changed the conditions for rows 21-28. 

I'm not a programmer, sorry; so I'm probably not being explicit enough. Hopefully the new wording is more explicit.

**Comment by lake44me** [comment]
Thanks, that helps.  So, for AAP/AP for 130 or 240, you include the subfields a, d, g, k, n, p, t.

"t" is not currently (and is not mentioned as formerly) a subfield for either 130 or 240 so you could omit it from the list.  (It is found in 700, 710, 711 name/title added entries with 2nd indicator 2, and $g does need to be positionally mapped as part of the name or title portion of 710 depending on whether it is ordered before or after the $t title subfield - mainly in the case of treaties).

**Comment by tmqdeborah** [comment]
@lake44me, $t might not be listed for 130 and 240 in the OCLC Bib Formats manual, but it is listed in the MARC Bibliographic manual, although it is given with a note "Subfield $t is unlikely to be used in an X30 field":
- 130: https://www.loc.gov/marc/bibliographic/bdx30.html

However, it see that subfield $t is not listed for a [240](https://www.loc.gov/marc/bibliographic/bd240.html). So, thanks for catching that, and I have removed it from the 240. 

Also, I rechecked the 700-711 title portion subfields and found a major copy and paste error on my part and have fixed it. Now, e.g., 700 *# $abcdjqu +g (if before $t) + knpt + g (if after $t), etc.

So, thanks for looking so closely and please let me know if you find anything else.

**Comment by lake44me** [comment]
@tmqdeborah and thanks for pointing out the $t defined for 130.  @JianPLee may be interested in this thread since she appears to be reviewing 130.

Since I am reviewing 710 mapping table as well as 240 I will consult the title portion section.

@CECSpecialistI Are we still expected to update the mapping spreadsheets for 130, 240, etc. to align with this table (once it is "done", approved or whatever)?  This could be simplified if, instead of transform notes, we just reference the table.

**Comment by tmqdeborah** [comment]
@AdamSchiff and @lake44me, for the 'has date of work' elements (rows 54-56) in the [Headings Attributes Table](https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=1632134046#gid=1632134046
), I was going by:
- coding that I found in <[m2r-relators.xsl](https://github.com/uwlib-cams/MARC2RDA/blob/f55273315da933d1cf192739b968c33078d7ea39/Working%20Documents/transformationCode/m2r-relators.xsl#L1275) at line 1275 for $n coding
- and this the LC Bib manual "[X30 - Uniform Titles-General Information](https://www.loc.gov/marc/bibliographic/bdx30.html)":

    Under $a
    Date added parenthetically to distinguish between identical uniform titles is not separately subfield coded.
    130 | 0#$aKing Kong (1933)
    130 | 0#$aKing Kong (1976)
    
    Under $n
    In music uniform titles, the serial, opus, or thematic index number, or a date used to distinguish one work from another, is contained in subfield $n.

I then also found coding for P10219 (date of work) for $f, without looking for the date qualifier: '\([12]\d\d\d\)'; so I added that qualifier as condition1. IOW, although $f is used for date of expression, if the date is enclosed in parentheses, then it might be date of work.

However, after our discussion in the meeting today, I have looked at a number of examples, found using the following logic:

    AND 130$a=\([1-2][0-9][0-9][0-9]\) [Case=Y] [RegEx] , OR 240$a=\([1-2][0-9][0-9][0-9]\) [Case=Y] [RegEx] , NOT 245$a=\([1-2][0-9][0-9][0-9]\) [Case=Y] [RegEx] , NOT 000/07=s [Case=Y] 

These examples exemplify a number of situations:
- the date might be a date of work (there are a considerable number of similar cases):
240 10$aConstitution (1876)
245 10$aTexas Constitution /$cprepared by the Research Division of the Texas Legislative Council.
250   $aRev. Jan. 2000.
260   $aAustin, Tex. :$bThe Council,$c[2000]
500   $a"January 2000."

110 1 $aEritrea.
240 10$aQewåam (1997).$lEnglish
260   $a[Asmara? :$bs.n.,$c1997]

- the date might not be a date of work (there are are considerable number of similar cases)
130 0 $aCensus of agriculture (1997).$n1,$pGeographic area series.
245 10$a1997 census of agriculture.$n1,$pGeographic area series$h[computer file].
260   $a[Washington, D.C.] :$bU.S. Dept. of Agriculture, National Agricultural Statistics Services,$c1999.
500   $a"Issued March 1999."
    ---- Is the work the census itself (done in 1997) or the compilation and publication of the results of the census (1999)

- the date might be a date of expression
240 10$aèAgypten (1984).$lEnglish
245 14$aThe search for God in ancient Egypt /$cby Jan Assmann ; translated from the German by David Lorton.
250   $a1st English-language ed., with revisions and additions.
260   $aIthaca :$bCornell University Press,$c2001.

Example files:

[Heading Fields Attributes_$a date of work examples.txt](https://github.com/user-attachments/files/19784006/Heading.Fields.Attributes_.a.date.of.work.examples.txt)

[Heading Fields Attributes_$n date of work examples.txt](https://github.com/user-attachments/files/19784009/Heading.Fields.Attributes_.n.date.of.work.examples.txt)

Based on these findings, I now prepared to agree that it might be best to give up on this method of finding a 'date of work' attribute this way for $a and $t. I can only hope that when catalogers switch to official RDA they will add an appropriate 'date of work', 'date of expression', or 'date of manifestation' attribute whenever they add a date to an access point. Imagine how useful it would be to a user to know what a date in an access point might mean.

I have struck out the rows for $a and $t and added a comment to indicate this decision was made.

Should I do the same for the rows for $n and $f?

If so, then the coding described above will have to be removed.

**Comment by AdamSchiff** [comment]
For music, I agree that $n can sometimes hold a date of work.  Would it be easy to differentiate dates from serial, opus, or thematic index numbers that also get recorded in $n?

Assmann appears to have written two different works with preferred title Ägypten:

100 1 Assmann, Jan.
240 10 Ägypten (1984)
245 10 Ägypten : ǂb Theologie und Frömmigkeit einer frühen Hochkultur / ǂc Jan Assmann.
260  Stuttgart : ǂb Kohlhammer, ǂc ©1984.

100 1 Assmann, Jan.
240 10 Ägypten (1996)
245 10 Ägypten : ǂb eine Sinngeschichte / ǂc Jan Assmann.
260  München : ǂb Hanser, ǂc ©1996.

So I believe that (1984) in the 240 is a date of work, not a date of expression.  For it to be date of expression, it should be in subfield $f and not in parentheses.

The date in Census of agriculture (1997) looks like a date of creation of work, but I agree that it may actually represent (in original RDA terms) other distinguishing characteristic of work, since it reflects when the census was taken but not when the works produced from it were actually created.

This morning, I was thinking about titles that include dates that are part of the preferred title.  For example this AACR2 authority for two expressions:

100 1#  Leyden, Michael J. ǂq (Michael Joseph), ǂd 1950- ǂt China-Tianjin postal and cultural history (1878-1947). ǂl Chinese & English

If we converted it to RDA, of course we'd need two NARs:
100 1#  Leyden, Michael J. ǂq (Michael Joseph), ǂd 1950- ǂt China-Tianjin postal and cultural history (1878-1947). ǂl Chinese  
100 1#  Leyden, Michael J. ǂq (Michael Joseph), ǂd 1950- ǂt China-Tianjin postal and cultural history (1878-1947). ǂl English

Here are some others in the NAF, some works, some expressions, but the dates are part of the preferred title of the work:

100 1#  Adorno, Theodor W., ǂd 1903-1969. ǂt Zur Lehre von der Geschichte und von der Freiheit (1964/65). ǂl English

100 1#  Alonso, Carlos, ǂc OSA. ǂt Agustinos en la costa Suahili (1598-1698). ǂl English

100 1#  Linteau, Paul André. ǂt Histoire du Québec contemporain. ǂn 1. ǂp De la Confédération à la crise (1867-1929). ǂl English

100 1#  Morenés y Mariátegui, Carlos. ǂt Historia del Coto de Doña Ana (1865-1985). ǂl English

100 1#  Walewski, Antoni, ǂd 1805-1876. ǂt Historya wyzwolenia Polski za panowania Jana Kaźmierza (1655-1660)

130 #0  Studies in European urban history (1100-1800)

130 #0  Studies in the history of daily life (800-1600)

**Comment by AdamSchiff** [comment]
If a record is coded as 040 $e rda, then I think it's safe to assume that $f represents a date of expression.  With pre-RDA such as AACR2 and earlier, all bets are off.

**Comment by CECSpecialistI** [comment]
> [@tmqdeborah](https://github.com/tmqdeborah) and thanks for pointing out the $t defined for 130. [@JianPLee](https://github.com/JianPLee) may be interested in this thread since she appears to be reviewing 130.
> 
> Since I am reviewing 710 mapping table as well as 240 I will consult the title portion section.
> 
> [@CECSpecialistI](https://github.com/CECSpecialistI) Are we still expected to update the mapping spreadsheets for 130, 240, etc. to align with this table (once it is "done", approved or whatever)? This could be simplified if, instead of transform notes, we just reference the table.

Yes, instead of transform notes I think we are just going to be referencing this in the table. We're pushing this until June/July and dealing with it during the documentation/deliverables phase. We'll shorthand it in the spreadsheets for sure.

**Comment by tmqdeborah** [comment]
The [notes ](https://github.com/uwlib-cams/MARC2RDA/wiki/2025-Meeting-Minutes#attributes-table-25)from the discussion on April 16, 2025 re. the [Attributes table](https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=1632134046#gid=1632134046
), say "700/710/711 - Deborah will double check".
I have done so, and have decided that since the coding instructions for 700-730 *2 fields for additional aggregated works are not needed until Phase II, they are not needed at this point; so, I removed them.

**Comment by tmqdeborah** [comment]
> For music, I agree that $n can sometimes hold a date of work. Would it be easy to differentiate dates from serial, opus, or thematic index numbers that also get recorded in $n?

We would look for the pattern: If $n = '\([1-2][0-9][0-9][0-9]\)'
> 
> Assmann appears to have written two different works with preferred title Ägypten:
> 
> 100 1 Assmann, Jan. 240 10 Ägypten (1984) 245 10 Ägypten : ǂb Theologie und Frömmigkeit einer frühen Hochkultur / ǂc Jan Assmann. 260 Stuttgart : ǂb Kohlhammer, ǂc ©1984.
> 
> 100 1 Assmann, Jan. 240 10 Ägypten (1996) 245 10 Ägypten : ǂb eine Sinngeschichte / ǂc Jan Assmann. 260 München : ǂb Hanser, ǂc ©1996.
> 
> So I believe that (1984) in the 240 is a date of work, not a date of expression. For it to be date of expression, it should be in subfield $f and not in parentheses.
> 
> The date in Census of agriculture (1997) looks like a date of creation of work, but I agree that it may actually represent (in original RDA terms) other distinguishing characteristic of work, since it reflects when the census was taken but not when the works produced from it were actually created.
> 
> This morning, I was thinking about titles that include dates that are part of the preferred title. For example this AACR2 authority for two expressions:
> 
> 100 1# Leyden, Michael J. ǂq (Michael Joseph), ǂd 1950- ǂt China-Tianjin postal and cultural history (1878-1947). ǂl Chinese & English
> 
> If we converted it to RDA, of course we'd need two NARs: 100 1# Leyden, Michael J. ǂq (Michael Joseph), ǂd 1950- ǂt China-Tianjin postal and cultural history (1878-1947). ǂl Chinese 100 1# Leyden, Michael J. ǂq (Michael Joseph), ǂd 1950- ǂt China-Tianjin postal and cultural history (1878-1947). ǂl English
> 
> Here are some others in the NAF, some works, some expressions, but the dates are part of the preferred title of the work:
> 
> 100 1# Adorno, Theodor W., ǂd 1903-1969. ǂt Zur Lehre von der Geschichte und von der Freiheit (1964/65). ǂl English
> 
> 100 1# Alonso, Carlos, ǂc OSA. ǂt Agustinos en la costa Suahili (1598-1698). ǂl English
> 
> 100 1# Linteau, Paul André. ǂt Histoire du Québec contemporain. ǂn 1. ǂp De la Confédération à la crise (1867-1929). ǂl English
> 
> 100 1# Morenés y Mariátegui, Carlos. ǂt Historia del Coto de Doña Ana (1865-1985). ǂl English
> 
> 100 1# Walewski, Antoni, ǂd 1805-1876. ǂt Historya wyzwolenia Polski za panowania Jana Kaźmierza (1655-1660)
> 
> 130 #0 Studies in European urban history (1100-1800)
> 
> 130 #0 Studies in the history of daily life (800-1600)

Based on your persuasive argument, Adam, I have struck-through the instructions for treating dates in $a and $t title portions of an access point as reliable sources for 'date of work'.

I have retained the instructions for mapping 'dates' (i.e., numbers that appear to be dates) that are found in parentheses in $n and $f.

See the attached examples: 
Heading Fields Attributes_$n date of work examples.txt

[Heading Fields Attributes_$n date of work examples.txt](https://github.com/user-attachments/files/19870277/Heading.Fields.Attributes_.n.date.of.work.examples.txt)

Heading Fields Attributes_$f date of work examples.txt
[Heading Fields Attributes_subfield f_date of work examples.txt](https://github.com/user-attachments/files/19876730/Heading.Fields.Attributes_subfield.f_date.of.work.examples.txt)

The [notes ](https://github.com/uwlib-cams/MARC2RDA/wiki/2025-Meeting-Minutes#attributes-table-25)from the discussion on April 16, 2025 re. the [Attributes table](https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=1632134046#gid=1632134046), say "Date of work: sometimes year as parenthetical in title (historic, scientific, technical works) Deborah will pull some examples" The above is my response re. this issue.

**Comment by AdamSchiff** [comment]
Those are two really weird looking access points with $f:

Revue hellénique de droit international.$n1. année, no. 4$f(1948).

Vernon's city of Sudbury and town of Copper Cliff (Ontario) directory$f(1960)

The first one appears to be incorrectly formulated to represent an issue of a periodical.  It should be formulated similarly to the way a series access point would be, so I don't think the year should be given at all.  And the second one is just odd too.  If it's a work qualifier, it shouldn't be in $f.  If it's an expression addition, it shouldn't be in parentheses.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by tmqdeborah** [comment]
> Those are two really weird looking access points with $f:

So, I guess that means we shouldn't look for dates inside parentheses in $f.

**Comment by tmqdeborah** [comment]
I have just found 209 records (in my LC file of 5.6mil records) with $t and other work and expression subfields in a 1XX. See attached file and a few examples below:

010   $a   76610561 
110 1 $aUnited States.$tTreaties, etc.$gYugoslavia,$dNovember 18, 1970. --common pattern

010   $a   86040479 
100 1 $aZiefert, Harriet.$tHello reading!

010   $a   87675352 
110 1 $aUnited States.$bArmy.$bCorps of Engineers.$bChicago District.$tCharts of the Illinois Waterway.

This is allowed by MARC Bib, but I didn't think that anyone was doing it.

It is particularly interesting when a 240 is also added:
010   $a   73012456 
110 1 $aTonga.$tLaws, etc. (Law of Tonga : 1967)
240 10$aLaw of the Government of Tonga

Do we need to add this situation to the coding instructions for APs and AAPs for 1XX headings in the [table](https://docs.google.com/spreadsheets/d/1k-ZevqjVntyqyGZ89IGma5j71ySLYZabGrn4Ls5GM3E/edit?gid=1632134046#gid=1632134046)?

**100 ** $abcjqu +dg (if before $t) + $knpt + dg (if after $t and before 'flmors')
110 ** $abcu + dgn (if before $t) + $kpt + dgn (if after $t and before 'flmors')
111 ** $acequ + dgn (if before $t) + $kpt + dgn (if after $t and before 'flmors'**)

100 ** $abcjqu +g (if before $t) + 240 $adknp + g (if before 'flmors')
110** $abcu + dgn (if before $t) + 240 $adknp + g (if before 'flmors')
111** $acequ + dgn (if before $t) + 240 $adknp + g (if before 'flmors')
    _note that have have to code for the second block above (+240) but if we code for both the 100 with title subfields and the 100 + 240, then we will get two AP/AAPs if they both appear as shown in the 240 example above_

Please advise.

------------------------------------------

**Added on 2025-05-07**: Decision: Add 100, 110, and 111 fields with title subfields (e.g., $t) to the table--as per the [2025-05-07 meeting](https://github.com/uwlib-cams/MARC2RDA/wiki/2025-Meeting-Minutes#headings-attributes-table-update-30)

And I have attached a file of the LC records in question.

Note that it also includes examples of $t followed by title subfields in the file. The coding is currently set to include $t in an AP|AAP if present. $t values will not also be added as an attribute; so, I think it is safe to map them, even when it is an obvious error.
130 0 $aBible.$lEnglish.$tNew World.$f1970. -- should be $s (Version)?
130 0 $aSchatzkammer der Optik.$tEnglish -- should be $f (Language)?
110 2 $aBlink-182 (Musical group)$tprf -- should be $4 (relator code)?
130 0 $aVedas.$tAtharvaveda.$lEnglish & Sanskrit. --ok?


[1XX$t_LC_examples.txt](https://github.com/user-attachments/files/20091986/1XX.t_LC_examples.txt)
